pywbem/_cim_operations.py

```diff
@@ -900,7 +918,7 @@ def scheme(self):
     def host(self):
         """
         :term:`unicode string`: Normalized host and port number of the WBEM
-        server, in the format ``host:port``.
+        server or WBEM listener, in the format ``host:port``.
 
         Default port numbers (5988 for http and 5989 for https) have been
```

Comment:
These port numbers are for servers only

Reply:
The `host` property will always contain `host:port`, even for listeners, since the property has the result of URL parsing, with missing port number defaulted dependent on the scheme.

For listeners, we now state that a port should be specified in the URL, and regardless of whether that is done, the `host`property will have a port.

Reply:
No Issiue DONE